README.md

```diff
@@ -1 +1,34 @@
 # nl-covid19-notification-app-dashboard
+
+## What is GitHub Pages
+
+[GitHub Pages](https://pages.github.com/) are public webpages hosted directly through the GitHub repository.
+
+## What is Jekyll
+
+GitHub Pages support the static site generator [Jekyll](https://jekyllrb.com/).
+Jekyll supports [Markdown](https://daringfireball.net/projects/markdown/), [Liquid](https://github.com/Shopify/liquid/wiki), HTML and CSS to create a complete static website.
+By using the Liquid templating language, content can be stored in Markdown.
+
+## How to add a feature
+
+To add a feature, create a Markdown file in `_features` with the following template:
+```md
+---
+title: The title
+summary: Short summary
+status: in-progress
+---
+The Markdown content goes here, this can contain <code>HTML</code>.
+```
+For the status field the values `failed`, `in-progress` and `checked` can be used.
+Please prefix your file with 3 zero's and a dash (`000-`), it will be renamed to have an incremental number when we merge it.
+
+## How to test GitHub Pages locally
+
+The [`starefossen/github-pages`](https://hub.docker.com/r/starefossen/github-pages) is a Alpine Docker image for running GitHub Pages / Jekyll projects locally while developing of only 70 MB.
+You need to do is to mount the pages in a volume under `/usr/src/app` like this:
+```bash
+docker run -it --rm -v "$PWD":/usr/src/app -p "4000:4000" starefossen/github-pages
+```
+The Jekyll page will be available on `http://localhost:4000`.
```

Comment:
If choosing to suggest docker here, it would probably be helpful to provide a link to a page that explains what docker is and how to download/install it.

Reply:
Patches are welcome :slightly_smiling_face:
Also I only tested this on Linux, `$PWD` probably only works in Powershell and not plain `cmd` on Windows?

Reply:
As for the current working dir, I believe this one is correct https://stackoverflow.com/a/921762/95397

Reply:
Can you test it? :slightly_smiling_face: Will `%cd%` expanding also work inside a string parameter? I believe `$PWD` [does work in Powershell](https://superuser.com/a/295036).

Reply:
Sure, let me grab a windows machine 👍

Reply:
So yeah, `%cd%` works in `cmd` (the top window) and `$PWD` works in Powershell (the bottom window):

![IMG_1735](https://user-images.githubusercontent.com/2320/83139161-35967400-a0ec-11ea-825e-ce7bdb3105e0.jpeg)

Probably sufficient to suggest the use of Powershell on Windows.

Reply:
Fixed in https://github.com/minvws/nl-covid19-notification-app-dashboard/pull/3/commits/e7a12f595500aebe53c450d46f611f9aa4c479e8